src/EhsnPlugin/DataModel/ParsedEhsnLevelSurvey.cs

```diff
@@ -35,14 +35,16 @@ public class ParsedEhsnLevelSurvey
             LevelSummaryRows = new List<EHSNLevelNotesLevelChecksSummaryTableRow>(summaryTableRows);
         }
 
-        private const string PrimaryPrefix = "*";
+        private const string PrimarySuffix = "*";
+        private const string PrimaryPrefix = "**";
 
         public static string SanitizeBenchmarkName(string value)
         {
-            if (value == null || !value.Contains(PrimaryPrefix))
-                return value;
-
-            return value.Replace("*", string.Empty).Trim();
+            if (value.StartsWith(PrimaryPrefix))
+                return value.Substring(PrimaryPrefix.Length).Trim();
+            if (value.EndsWith(PrimarySuffix))
+                return value.TrimEnd('*').Trim();
+            return value;
```

Comment:
@yanxuYX I'm finally getting around to the review! (Sorry, I've been swamped).

Not a big deal at all, but if you just need to remove leading and trailing asterisks from the benchnames, then you can do that with a single line in C#, using the null-aware method call (the `?.` operator):

```C#
public static string SanitizeBenchmarkName(string value)
{
    return value?.Trim('*');
}
```